reactive/kubernetes_master.py

```diff
@@ -620,8 +619,11 @@ def setup_leader_authentication():
     # Try first to fetch data from an old leadership broadcast.
     if not get_keys_from_leader(keys) \
             or is_state('reconfigure.authentication.setup'):
+        if not os.path.isfile(basic_auth):
+            kubernetes_master.deprecate_auth_file(basic_auth)
```

Comment:
Why are we replacing this with a deprecation stub rather than making it non-blocking as mentioned in the bug?

Reply:
@johnsca I called an audible.  I was thinking it might be nice for users that are familiar with CK to have stubbed out auth.csv with a note rather than them just being gone.  I don't feel strongly either way, so if you'd prefer these file being non-blocking, I can switch that in a jiff.

Reply:
This approach is definitely a smaller delta.  I think that trying to actually avoid the files might have knock-on effects like I thought would happen in my comment below, which this approach probably actually avoids.